core/chains/evm/config/toml/config.go

```diff
@@ -412,8 +412,16 @@ func (c *Chain) ValidateConfig() (err error) {
 					err = multierr.Append(err, commonconfig.ErrInvalid{Name: "Transactions.AutoPurge.DetectionApiUrl", Value: c.Transactions.AutoPurge.DetectionApiUrl.Scheme, Msg: "must be http or https"})
 				}
 			}
-		case chaintype.ChainZkEvm:
-			// No other configs are needed
+		case chaintype.ChainZkEvm, chaintype.ChainXLayer:
```

Comment:
Why does this AutoPurge.MinAttempts check with GasEstimator.BumpThreshold need to be conditional only for chaintype.ChainZkEvm, chaintype.ChainXLayer?

Seems like unnecessary, and just adds more complexity. Someone could try to set MinAttempts for another chain, and cause problems, because this validation wasn't enabled for that chain.

Reply:
Isn't the validation still performed for all other ChainTypes in the `default` case?
https://github.com/smartcontractkit/chainlink/pull/14132/files/0131d67e7cfdddc53c05c3330ea8d67e1645dde8#diff-7cc03cd5adaef6b2b6e25f221cf16163cbd44f0b8f2bf188febaf01389a7bf9eL429-L430

Reply:
The reason for the chain specific validations is because different combination of configs are needed for the different detection methods. I didn't want to broadly validate `MinAttempts` if it's accidentally set for a chain that ignores it and it causes unnecessarily problems. Also Friedemann is correct that we validate this already for any chain in the default case because all other chains would use the heuristic.